homeassistant/components/cloud/account_link.py

```diff
@@ -45,7 +45,7 @@ def _is_older(version: str) -> bool:
         return False
 
     try:
-        version_parts = [int(val) for val in version_parts]
+        version_parts_int = [int(val) for val in version_parts]
```

Comment:
This entire function can be replaced with `AwesomeVersion`
<https://github.com/home-assistant/core/pull/55449>

Reply:
Thanks, rebased